### ISSUE TITLE
Show usage numbers in blocklist admin for unlisted add-ons

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#13123]
### Describe the problem and steps to reproduce it:

Unlisted add-ons (with an `update_url`) do not ping AMO for updates and thus AMO does not have a record of their daily users count. The blocklist admin needs to show that information though, as it's critical for a blocklist reviewer to assess and sign-off on a block.

Usage information on aforementioned add-ons is available in telemetry, which the new stats processing pipeline will also be based on.

So for this feature, we either need to make a call to telemetry or we'd need to expedite the stats processing changes, so those telemetry numbers are fed to AMO.

**Comment by jvillalobos** [comment]
The intention is to wait for the stats processing changes, which are scheduled for Q1.

**Comment by diox** [comment]
Project for the stats processing changes: https://github.com/orgs/mozilla/projects/116 (it's empty, but basically the whole project is blocking this).

**Comment by eviljeff** [comment]
Blocked by #14428

**Comment by ioanarusiczki** [comment]
@eviljeff For an add-on with data, I see the number of users being available as for [stats](https://addons-dev.allizom.org/en-US/firefox/addon/some-addon/statistics/beta/?last=30).

![users](https://user-images.githubusercontent.com/33448286/85135033-164fba00-b246-11ea-9709-20cc51023372.png)

I've noticed that the block form accessed from the review page looks different from the one created directly from the admin. 
I could not block the addon straight from the review page so I tried to add the block from the admin and on the second page the min and max version won't display the versions. This is the [addon ](https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/some-addon) and it has one unlisted version and two are listed.

**Comment by ioanarusiczki** [comment]
@eviljeff So these errors were caused because I replaced the initial guid of the addon with another one in order to have stats data from prod.
I can't tell if that would be a real scenario on production, an admin to change the GUID.

**Comment by eviljeff** [comment]
If the guid was changed on the Addon there would be an inconsistency like you've found (the link redirected to form with the wrong guid filled in).  
So, it's not related to this issue with usage stats (or really blocklists in general - it's being able to edit an addon's guid and the AddonGUID model not being updated too)

Admins changing guids is a rare and dangerous thing to do so I think it's unlikely there would be any crossover with blocking them.

Though I suppose it's an edge case we should consider and either handle that when the admin updates the guid, or prevent updating guids entirely.  @ioanarusiczki  Can you log a follow-up issue?

**Comment by ioanarusiczki** [comment]
@eviljeff I filed it https://github.com/mozilla/addons-server/issues/14700

As for stats I think it should be alright. [Users](https://addons-internal.dev.mozaws.net/en-US/admin/models/blocklist/block/26223/change/) show up for an edit block too which is equal to beta [ADU](https://addons-dev.allizom.org/en-US/firefox/addon/amo-info-with-extra-dirs/statistics/beta/?last=7)
